developer/src/kmcmplib/src/Compiler.cpp

```diff
@@ -1731,7 +1734,7 @@ KMX_DWORD GetXString(PFILE_KEYBOARD fk, PKMX_WCHAR str, KMX_WCHAR const * token,
 }
 
 KMX_DWORD GetXStringImpl(PKMX_WCHAR tstr, PFILE_KEYBOARD fk, PKMX_WCHAR str, KMX_WCHAR const * token,
-  PKMX_WCHAR output, int max, int offset, PKMX_WCHAR *newp, int isVKey, int isUnicode
+  PKMX_WCHAR output, int max, int offset, PKMX_WCHAR *newp, int /*isVKey*/, int isUnicode
```

Comment:
```suggestion
  PKMX_WCHAR output, int max, int offset, PKMX_WCHAR *newp, int isUnicode
```

And fix callers...

Reply:
There's only one caller, however `GetXString()` has lots of callers (38, scattered) with both TRUE and FALSE values here.  Is this related to the commented out lines somehow?

```c
    case '[':  type = 11; break;	// start of vkey section
    //case ']':  type = 12; break;	// end of vkey section
    //case 'K':  type = 13; break;	// virtual key name or "key"
```

Reply:
GetXStringImpl fixed, GetXString is not (above comment)